docs/config.rst

```diff
@@ -1,9 +1,9 @@
 .. be in -*- rst -*- mode!
 
-tox configuration specification
+tox Configuration Specification
```

Comment:
I see no reason to change anything on this line

Reply:
Actually, I tried to make first letter of each word of document title as caps. I've seen many people follows this convention. If you want to revert these changes, I'm more than happy to make suggested changes.

Reply:
Yes, I'd like to revert.